resources/multisite_drupal_standard.make

```diff
@@ -526,7 +526,9 @@ projects[piwik][subdir] = "contrib"
 projects[piwik][version] = "2.9"
 
 projects[plupload][subdir] = "contrib"
-projects[plupload][version] = "1.7"
+projects[plupload][download][branch] = 7.x-1.x
+projects[plupload][download][revision] = bba974c6f3224346a1acae4181a700b55129e6e1
```

Comment:
@drupol can you document this section of the make file, describing why we use the particular revision? It will be easier then for our future selfs and for others to know why this was changed.